README.md

```diff
@@ -4,16 +4,15 @@ Sopel module that fetches info about torrent links
 ## Current status
 
 The current support is very limited, just a proof of doing something. It fetches
-only from Nyaa, and then only a very few details. More information (and possibly
+only from Nyaa.si and Anidex.info, and then only a very few details. More information (and possibly
 other supported sites) will come later.
 
 ## Requirements
 The torrentinfo module relies on the following Python components:
 
-* `bleach`
-* `cssselect`
-* `etree` from `lxml`
+* `lxml`
 * `requests`
+* `bleach`
```

Comment:
With the changes you made to drop PyQuery, the requirements list from `master` will do (though specifying just `lxml` is OK). `cssselect` was added since this PR was opened, and it should be listed separately because it is not technically part of `lxml` any more.

Reply:
I should have been more explicit earlier: The requirements list should be alphabetized too.